### ISSUE TITLE
Problème de build sur Mac Arm64 (M1)

### ISSUE THREAD

**Issue by MisterGado** [openMVG/openMVG#1864]
Bonjour,

J'essaye d'installer openMVG sur mon ordinateur (MacBook Pro avec la nouvelle architecture Apple ARM64).
La majorité du build se passe sans accroc mais à environ 75%, une erreur se produit (cf pièce jointe). Si j'essaye de faire make -k (keep going), je vois que la même erreur se répète un certain nombre de fois ensuite (je crois que c'est la seule).
Les fichiers touchés semblent être nécessaires (par exemple openMVG_main_SfMInit_ImageListing qui est toujours le premier appelé dans les scripts que j'ai pu voir).

Merci d'avance pour votre aide !

Alexis Gado.

<img width="566" alt="Capture d’écran 2021-03-07 à 17 19 17" src="https://user-images.githubusercontent.com/56356237/110247175-34dc1980-7f6b-11eb-8b45-0ba2e8042a58.png">

PS : I can edit this post and translate it in English if you prefer.

**Comment by pmoulon** [comment]
Hello, Bonjour,

It seems that you are using OpenMVG and using the internal png library.
Can you try install png library on your mac, or download one and install it and try again?

Our cake build for libpng seems not to cover all the subtleties of ARM for Mac

**Comment by MisterGado** [comment]
Thank you for your reactivity ! 
I installed libpng on my computer but cmake keeps using the internal library (even when I precise -dcmake_prefix_path=/path/to/libpng). Am I missing something ? 
<img width="494" alt="Capture d’écran 2021-03-08 à 02 26 28" src="https://user-images.githubusercontent.com/56356237/110263015-b5276c80-7fb5-11eb-8af4-cc51df454f8e.png">

**Comment by pmoulon** [comment]
OpenMVG should pick up if libpng is installed on your computer.
Did you try empty the cmake cache and relaunch cmake again?

**Comment by MisterGado** [comment]
Unfortunately it keeps pointing to the internal library. I will try again tomorrow.

**Comment by pmoulon** [comment]
What I wonder is why this line is not using your compiled and installed libpng https://github.com/openMVG/openMVG/blob/develop/src/CMakeLists.txt#L169

-> https://cmake.org/cmake/help/latest/module/FindPNG.html

**Comment by MisterGado** [comment]
Okay I managed to have libpng detected but I am facing another issue (potentially from another cause).
The make step crashes quasi instantly.

Thank you very much for your help !

<img width="763" alt="Capture d’écran 2021-03-08 à 14 53 18" src="https://user-images.githubusercontent.com/56356237/110330622-43373d80-801e-11eb-8886-62e4a5844745.png">

**Comment by MisterGado** [comment]
I tried some new things today,
I Installed coinUtils, then I had the same fatal error while building Lemon, so I installed lemon.
Then I started to get new errors : SSE2 register return with SSE2 disabled. I m quiet lost there.

Have a good day !

<img width="790" alt="Capture d’écran 2021-03-10 à 18 12 16" src="https://user-images.githubusercontent.com/56356237/110669436-9e0e9780-81cc-11eb-8a75-1850654e01e5.png">
